components/post_view/post_list_row/index.ts

```diff
@@ -17,8 +17,8 @@ import {GlobalState} from 'types/store';
 import PostListRow from './post_list_row';
 
 type Props = {
-    post: Post;
```

Comment:
Do we actually need to declare these props in this function? It doesn't look like we need them in `mapStateToProps`, and getting rid of `ownProps` from there will actually be a slight performance improvement since React-Redux treats components with and without that slightly differently